source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -28,5 +27,78 @@ more naturally readable.
 Operations
 ----------
 
+The following sections provide information and examples for
+aggregation expression operations available in the driver.
+The operations are categorized by purpose and functionality.
+
+.. important::
+
+   The driver generates a Query API expression that may be different
+   from the Query API expression provided in each example. However,
+   the expressions are equivalent when evaluated.
+
+Boolean Operations
+~~~~~~~~~~~~~~~~~~
+
+You can construct a boolean operation from at least one value of type
+``MqlBoolean`` and one or more boolean aggregation methods.
```

Comment:
In this Java API, for something like `oneValue.not()`, the "one value" is the `MqlBoolean` `oneValue` that the method is invoked on.

Only one (strike "or more") of the methods below is used per operation. (I think perhaps we should avoid introducing a novel term "boolean aggregation methods".)

```suggestion
You can perform an operation on an ``MqlBoolean`` value using the methods below.
```

(Just for reference, especially in the context of Java: operations are performed, objects are constructed (when a constructor is involved), methods are invoked.)